Gemfile

```diff
@@ -33,6 +33,7 @@ group :system_tests do
     gem 'beaker-rspec',  :require => false
   end
   gem 'serverspec',    :require => false
+  gem 'master_manipulator', '~> 1.1',  :require => false
```

Comment:
I just discovered that the latest version of ```master_manipulator``` has an issue with PE 3.8.x. I would suggest pinning to the 1.1.2 version of the gem.